### ISSUE TITLE
A1:m with L1 loss.

### ISSUE THREAD

**Issue by leyuan-sun** [6550Zhao/Policy-World-Model#3]
Does this L1 loss for actions require action label (GT) during training?
`For training, Xt and Dt+1:t+n are supervised with cross-entropy loss, and A1:m with L1 loss. `

**Comment by 6550Zhao** [comment]
Yes, that's accurate! The L1 loss for A1:m requires ground-truth action labels during training, and cross-entropy loss is used to supervise Xt and Dt+1:t+n.
